web/list/notifications/index.php

```diff
@@ -6,7 +6,13 @@
 	// Data
 	exec(HESTIA_CMD . "v-list-user-notifications $user json", $output, $return_var);
 	$data = json_decode(implode("", $output), true);
+
+	function sort_priorty_id($element1, $element2) {
+		return $element2["PRIORITY"] - $element1["PRIORITY"];
+	}
 	$data = array_reverse($data, true);
+	usort($data, "sort_priorty_id");
```

Comment:
as of PHP>=8.1.0 we should use the new callable syntax
```php
usort($data, sort_priorty_id(...));
```
instead ^^ it's more idiomatic, and avoids some issues with namespace issues (albeit there's no namespace issue in this specific instance)